constitution.tex

```diff
@@ -487,7 +498,9 @@
 \hline
 Research and Development & 20\% \\
 \hline
-Social & 20\% \\
+Social & 15\% \\
```

Comment:
Are you sure you want this? Socials frequently runs out of budget

Reply:
PR already draws from Socials, and any public events or charity events will draw directly from this budget now instead of Socials'

Reply:
At it's creation, the PR adhoc was assigned a budget. What was that budget?

Reply:
Right now, PR's budget is $400. This change (going off of current spending info) would make the budget about $687. Because of the transition to using PR's budget for public events instead of Socials', it should work out that socials spends less and PR spends significantly more.

Reply:
> Because of the transition to using PR's budget for public events instead of Socials'
PR should not be using any of social's budget. When has this happened?

Reply:
PR has not, that was wrong. My previous statement is not affected though.

Reply:
I have no idea what you're trying to say

Reply:
I'm not sure what's confusing, but at this point it seems like you may have a semantic problem with the amount of money being allocated, which I will be happy to provide justification for at house meeting.